<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QgsWFSSensorLayerDialogBase</class>
 <widget class="QDialog" name="QgsWFSSensorLayerDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WFS sensor layers</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QToolButton" name="mAddRowButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../images/images.qrc">
       <normaloff>:/images/themes/default/mActionAdd.svg</normaloff>:/images/themes/default/mActionAdd.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QToolButton" name="mRemoveRowButton">
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../images/images.qrc">
       <normaloff>:/images/themes/default/mActionRemove.svg</normaloff>:/images/themes/default/mActionRemove.svg</iconset>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>370</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QTableWidget" name="mTableWidget">
     <column>
      <property name="text">
       <string>WFS url</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>SOS url</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>ID attribute</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Begin attribute</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>End attribute</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QDialogButtonBox" name="mButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>mButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>QgsWFSSensorLayerDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>QgsWFSSensorLayerDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
